pkgs/development/compilers/vlang/default.nix

```diff
@@ -97,6 +97,7 @@ stdenv.mkDerivation {
     $out/lib/v -v $out/lib/cmd/tools/vdoc
     $out/lib/v -v $out/lib/cmd/tools/vast
     $out/lib/v -v $out/lib/cmd/tools/vvet
+    $out/lib/v -v $out/lib/cmd/tools/vcreate
```

Comment:
A better description would be `vlang: add link to vcreate`

Reply:
Yes, i am editing it. Thank you for the feedback.